src/Widgets/Stats.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Lle\DashboardBundle\Widgets;
+
+use Lle\DashboardBundle\Form\StatsType;
+use Symfony\Component\Form\FormInterface;
+
+class Stats extends AbstractWidget
+{
+    public function __construct(private iterable $dataProvider)
+    {
+    }
+
+    public function render(): string
+    {
+        $providers = [];
+        $providers[] = '';
+        $providerConfs = [];
+        foreach ($this->dataProvider as $provider) {
+            $providers[] = get_class($provider);
+            $providerConfs[get_class($provider)] = $provider->getDataConf();
+        }
```

Comment:
ici tu fait plutot un 
$dataSources = [];
foreach ($this->dataProvider as $provider) {
   foreach($provider->getDataConf() as $conf) {
       $dataSources[] = $conf
   }
}

Reply:
du coup, t'a juste une déroulante simple.

Reply:
faut juste pour la key avoir un prefixe pour savoir quel provider appeler.